relay-config/src/redis.rs

```diff
@@ -273,35 +277,38 @@ pub(super) fn create_redis_pool(
     }
 }
 
-pub(super) fn create_redis_pools(
+/// Builds a [`RedisConfigsRef`] given a [`RedisConfigs`].
+///
+/// The returned configs contain more options for setting up Redis.
+pub(super) fn build_redis_configs(
     configs: &RedisConfigs,
     cpu_concurrency: u32,
-    max_pool_concurrency: u32,
-) -> RedisPoolConfigs {
-    // Default `max_connections` for the `project_configs` pool.
-    // In a unified config, this is used for all pools.
+    pool_concurrency: u32,
+) -> RedisConfigsRef<'_> {
+    // Default `max_connections` for the `project_configs` client.
+    // In a unified config, this is used for all clients.
     let project_configs_default_connections =
         std::cmp::max(cpu_concurrency * 2, DEFAULT_MIN_MAX_CONNECTIONS);
 
     // The number of default connections is equal to how many threads we have times the number of
     // futures we can concurrently drive times some leeway since we might use more connections.
-    let default_connections = cpu_concurrency * max_pool_concurrency * 2;
+    let default_connections = cpu_concurrency * pool_concurrency * 2;
```

Comment:
Is this still correct with the async client?

Reply:
Nope, that config parameter is not needed anymore. I was considering following up with a PR to clean up all the unused config params. https://github.com/getsentry/team-ingest/issues/682 this was the ticket I created.